ferry.go

```diff
@@ -443,7 +443,9 @@ func (f *Ferry) Initialize() (err error) {
 	f.BinlogStreamer = f.NewBinlogStreamer(f.SourceDB, f.Config.Source)
 
 	if !f.Config.SkipTargetVerification {
-		targetVerifier, err := NewTargetVerifier(f.TargetDB, f.StateTracker, f.NewBinlogStreamer(f.TargetDB, f.Config.Target))
+		targetBinlogStreamer := f.NewBinlogStreamer(f.TargetDB, f.Config.Target)
+		targetBinlogStreamer.LogTag = "tgt_binlog_streamer"
```

Comment:
This works, but I'd rather see an additional tag of `source`/`target` in addition to the `binlog_streamer` tag, unless we only get one tag, then potentially prepending it would make sense.

Reply:
I can tag with a hostname, but it maybe a bit verbose.. Let's go with this for now and change it later if we want to?